### DISCUSSION TITLE
Making quarto seeing portable MiKTeX installation

### DISCUSSION THREAD

**Discussion by psads-git** [quarto-dev/quarto-cli#9316] [Q&A] [answered]
### Description

How to make `quarto` aware of a `MiKTeX` portable installation?

I have searched for a solution, but found nothing. 

Thanks in advance!

**Comment by mcanouil** [comment, marked as answer]
As you would for any software on any platform, *i.e.*, make it discoverable in `PATH`.

Do note that, Quarto recommends the usage of TinyTex and cannot really help setting up properly your LaTeX distribution (missing packages, etc.).
So, for most of your (future) trouble, you'll have to go on more specialised places and use `keep-tex: true` to investigate the LaTeX issues you might encounter.

**Comment by psads-git** [comment]
I was speculating that there were some `quarto` config file for that, @mcanouil. Thanks!

**Reply by mcanouil** [reply]
That would not be very maintainable to have an option for every third party components while `PATH` is the common way to use/find tools.

The only (as far as I know) tools which you can set the path are R and Python using `QUARTO_R` and `QUARTO_PYTHON` environment variables.

**Comment by psads-git** [comment]
> That would not be very maintainable to have an option for every third party components while `PATH` is the common way to use/find tools.

That makes sense!
